traffic_ops/app/conf/cdn.conf

```diff
@@ -10,6 +10,20 @@
         "workers" : 12
     },
     "traffic_ops_golang" : {
+        "traffic_vault_backend": "",
```

Comment:
you might've miscommitted these changes again

Reply:
I think I might've misunderstood your previous comment. I thought you were referring to the values being committed, and not the keys. I get it now, will remove.